config/src/waypoint.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{format_err, Error, Result};
+use libra_crypto::HashValue;
+use libra_types::transaction::Version;
+use serde::{Deserialize, Serialize};
+use std::fmt::{Display, Formatter};
+use std::str::FromStr;
+
+// Waypoint format is '<version>:<hex encoding of the hash>'
```

Comment:
of what hash?

Reply:
Oh, I guess I should include the link to the documentation of a waypoint somewhere in the comment once it's well-organized. You can read more in the referenced issue. It's a hash of a LedgerInfo on the epoch boundary.

Reply:
why hex vs base64?

Reply:
Yea, right, base64 is better (shorter), will change.

Reply:
We should replace all these hex values in our config with base64!